server.rb

```diff
@@ -17,16 +17,22 @@
 EasyTranslate.api_key= ENV['TRANSLATE_API_KEY']
 @@users = {}
 
-def language(id)
-  languages = EasyTranslate::LANGUAGES.invert
-  choice = @@users[id].downcase
-  languages.each do |key, value|
-    if key == choice
-      return {to: value.to_sym}
-    end
+# Original Way I wrote the method
+  # def language(id)
+  #   languages = EasyTranslate::LANGUAGES.invert
+  #   choice = @@users[id][:language].downcase
+  #   languages.each do |key, value|
+  #     if key == choice
+  #       return {to: value.to_sym}
+  #     end
+  #   end
+  #   return nil
+  # end
+
+# Cleaner way to write the language method
+  def language(id)
+    EasyTranslate::LANGUAGES.invert[@@users[id][:language].downcase]
```

Comment:
I don't think this is cleaner >.< @harley what do you think

thisistoomanythingsononelineifyouaskme